railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_0.rb.tt

```diff
@@ -1,8 +1,8 @@
 # Be sure to restart your server when you modify this file.
 #
-# This file contains migration options to ease your Rails 6.0 upgrade.
-#
-# Once upgraded flip defaults one by one to migrate to the new default.
+# This file contains explicit, new configuration defaults to ease your Rails 6.0
```

Comment:
Don’t see the point of saying explicit or new. What does it mean that they’re explicit?

New is in the file name.